.github/workflows/ci.yaml

```diff
@@ -65,16 +65,24 @@ jobs:
     needs:
       - "rustfmt"
       - "markdown-lint"
+    strategy:
+      matrix:
+        rust:
+          - stable
+          - beta
+    # Prevent beta warnings from causing CI failure
+    continue-on-error: ${{ matrix.rust == 'beta' }}
```

Comment:
:question: So even if we have hard failures in beta we want the CI to pass?
The intent is not to condition on `-D warnings` for `beta`?

Reply:
Yeah, the idea is that if clippy is going to blow up in the future, it's good to know that, but it shouldn't block merging a PR (e.g. right now there's a clippy warning in beta about usage of `&format!()` as a needless borrow)